specification.md

```diff
@@ -1855,6 +1868,19 @@ for this connection.
 This field is optional: if omitted the client SHOULD NOT use a client-side certificate.
 This field can be used to perform a client certificate revocation/rotation.
 
+##### OpAMPConnectionSettings.heartbeat_interval_seconds
+
+If the ReportsHeartbeat capability is true, the Client MUST use the offered heartbeat
+interval to periodically send an AgentToServer message. At a minimum the instance_uid
+field MUST be set. It is recommended that the Agent also set ComponentHealth as well.
+An HTTP based-client MUST use the heartbeat interval as its polling interval.
+
+A heartbeat is used to keep a load balancer connection active and inform the server that
+the Agent is still alive and active. A server could use the heartbeat to make decisions about
+the liveness of the connected Agent.
+
+A default of a 30s should be used if not set by the OpAMPConnectionSettings.
```

Comment:
As brought up in #183, should we write a note about clients respecting values negotiated from the server or coming from a backpressure mechanism (eg. a Retry-After header?)